doc/modules/svm.rst

```diff
@@ -415,8 +417,9 @@ Tips on Practical Use
     applied to the test vector to obtain meaningful results. See section
     :ref:`preprocessing` for more details on scaling and normalization.
 
-  * Parameter ``nu`` in :class:`NuSVC`/:class:`OneClassSVM`/:class:`NuSVR`
-    approximates the fraction of training errors and support vectors.
+  * Parameter ``nu`` in :class:`NuSVC`, :class:`OneClassSVM`, :class:`NuSVR`,
+    and :class:`SVDD` approximates the fraction of training errors and support
+    vectors.
```

Comment:
Could you also add SVDD after OneClassSVM in **Randomness of the underlying implementations**? (the implemenations of OneClassSVM and SVDD are both not random)